crates/matrix-sdk-ui/src/room_list_service/room_list.rs

```diff
@@ -199,8 +199,6 @@ fn merge_stream_and_receiver(
                 biased;
 
                 diffs = raw_stream.next() => {
-                    tracing::error!("merge streams: receive diff");
```

Comment:
Please fold this commit (6132841f5c29e5984b663c46f7eac94cbb8f2a26) with the one that introduced the logs. There's no point in polluting our commit history by adding code in commits that's reverted in commits in the same PR (unless you plan to squash everything into a single commit at the end).